lib/components/CardGroup.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Card = ({ children }) => (
+  <div className="card-group__item">{children}</div>
+);
+
+Card.propTypes = {
+  children: PropTypes.any,
+};
+
+const CardGroup = ({ items }) => (
+  <div className="card-group">
+    {items.map(item => (
+      <Card>{item}</Card>
```

Comment:
Does it worth to have a new component for this?
Maybe we could just replace this with:
```suggestion
      <div className="card-group__item">{item}</div>
```

Reply:
Yes, I thinks this has more sense, because Card component it's too simple.